### PR TITLE
MM-11612 Add translation strings and add test case

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
add translations string for missing ids

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-11612

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -4682,6 +4682,14 @@
     "id": "model.user_access_token.is_valid.user_id.app_error",
     "translation": "Invalid user id"
   },
+  {
+    "id": "model.user.is_valid.email.app_error",
+    "translation": "Invalid email"
```
Comment (esethna) [inline i18n/en.json]: Why is it invalid? Can we add some more details so an end user has clear next steps to correct the username or email errors?

Comment (cpanato) [reply i18n/en.json]: like what @esethna this message is similar to another error message when the email is invalid.

Comment (cpanato) [reply i18n/en.json]: @esethna updated the text, please take a look and let me know what is your thoughts

File: i18n/en.json [hunk 2]
```diff
@@ -4694,6 +4694,14 @@
     "id": "model.user_access_token.is_valid.user_id.app_error",
     "translation": "Invalid user id"
   },
+  {
+    "id": "model.user.is_valid.email.app_error",
+    "translation": "Please enter a valid email. i.e. example@mattermost.com"
+  },
+  {
+    "id": "model.user.is_valid.username.app_error",
+    "translation": "Invalid username. Only characters and . _ - are accepted."
```
[the remark this answers is not in the document]
Comment (cpanato) [reply i18n/en.json]: done

File: i18n/en.json [hunk 3]
```diff
@@ -4694,6 +4694,14 @@
     "id": "model.user_access_token.is_valid.user_id.app_error",
     "translation": "Invalid user id"
   },
+  {
+    "id": "model.user.is_valid.email.app_error",
+    "translation": "Please enter a valid email. i.e. example@mattermost.com"
```
[the remark this answers is not in the document]
Comment (cpanato) [reply i18n/en.json]: done

[the remark this answers is not in the document]
Comment (cpanato) [reply i18n/en.json]: @esethna PTAL

**Comment by cpanato** [comment]
@esethna PTAL

**Pull request merged**